markdown/blog/2024-gsoc-wrap.md

```diff
@@ -0,0 +1,112 @@
+
+- **Focus on Quality Over Quantity:** Avoid making superficial contributions just to show activity. Mentors value meaningful, impactful contributions over sheer numbers. It doesn’t matter if you’ve only contributed to the website repo instead of the generator repo, what counts is the thoughtfulness and effort behind your work.
+
+- **Participate Beyond Code Contributions:** Being a successful GSoC contributor isn’t just about writing code. Engage in discussions, provide feedback on other contributions, and help answer questions from community members. Demonstrating your willingness to contribute in diverse ways shows your dedication to the community.
+
+- **Set Clear Project Timelines and Deliverables:** When preparing your GSoC proposal, ensure your timeline is realistic and achievable. Break your project into smaller milestones with clear deliverables for each phase. This demonstrates your organizational skills and helps mentors see your plan for success.
+
+- **Communicate Clearly and Effectively:** Good communication is key. Keep your mentors and the community updated on your progress, ask thoughtful questions, and don’t hesitate to seek guidance. Be proactive in addressing issues and responsive to feedback. Helping others in the community can also set you apart as a standout contributor.
+
+- **Seek and Act on Feedback:** Throughout the program, regularly request feedback from mentors and implement it into your work. Showing that you can learn, adapt, and improve is a critical skill that mentors highly value.
+
+By following these tips, you’ll not only increase your chances of being selected but also leave a lasting positive impact on the AsyncAPI community.
+
+## Closing Remark 
+
+We want to take a moment to extend our gratitude to the mentors who went out of their time and way to support and pass on their knowledge and forge meaningful relationships with our mentees. This program would not be possible without their unwavering dedication. 
+
+A special shoutout goes to Postman, who has always allowed us to push some of our project ideas under their umbrella over the last few years, while it's sad to see this come to an end, we love and appreciate the opportunities, and the community as a whole will never take it for granted. 
+
+And finally, a special thanks to Google for their belief in Free and Open Source Software(FOSS), for making this program possible every year as a show of their support to the open source ecosystem, and for allowing organizations like ours to bring passionate contributors and mentors to collaborate on a project all for the love of opensource. 
```

Comment:
I think Stephanie deserves a dedicated thank you for all the support we got from here, not only during Postman-departure but also now with stipends transfer - it is some high extra effort from her side that she doesn't have to do.